lib/scala/project-manager/src/main/scala/org/enso/projectmanager/boot/ProjectManager.scala

```diff
@@ -287,21 +312,25 @@ object ProjectManager extends ZIOAppDefault with LazyLogging {
     ZIO.succeed(SuccessExitCode)
   }
 
-  private def logServerStartup(): UIO[Unit] =
+  private def logServerStartup(processConfig: MainProcessConfig): UIO[Unit] =
     ZIO.succeed {
       logger.info(
         "Started server at {}:{}, press enter to kill server",
-        config.server.host,
-        config.server.port
+        processConfig.serverHost.getOrElse(config.server.host),
+        processConfig.serverPort.getOrElse(config.server.port)
```

Comment:
Please add overrides in the config (instead or in addition to the cli flags) https://github.com/enso-org/enso/blob/59329bd59aeacc4f60ccc4ef323dfaf6629c7710/lib/scala/project-manager/src/main/resources/application.conf#L35-L36

The same way it is done for the `network` https://github.com/enso-org/enso/blob/59329bd59aeacc4f60ccc4ef323dfaf6629c7710/lib/scala/project-manager/src/main/resources/application.conf#L23-L32

i.e the line 
```
   interface = ${?NETWORK_INTERFACE}
```
defines override of the `interface` value by the `NETWORK_INTERFACE` environment variable.

This way you can override config by providing the necessary environment variables without the need of adding extra CLI parameters.